examples/game_of_life.rs

```diff
@@ -210,18 +210,18 @@ fn update_board(
 fn pause_on_crouch(
     mut events: EventReader<SneakEvent>,
     mut board: ResMut<LifeBoard>,
-    mut clients: Query<&mut Client>,
+    mut layers: Query<&mut EntityLayer>,
 ) {
     for event in events.iter() {
         if event.state == SneakState::Start {
-            board.paused = !board.paused;
+            let mut layer = layers.single_mut();
 
-            for mut client in clients.iter_mut() {
-                if board.paused {
-                    client.set_action_bar("Paused".italic().color(Color::RED));
-                } else {
-                    client.set_action_bar("Playing".italic().color(Color::GREEN));
-                }
+            if board.playing {
+                board.playing = false;
+                layer.set_action_bar("Paused".italic().color(Color::RED));
+            } else {
+                board.playing = true;
```

Comment:
Why didn't you keep `board.paused = !board.paused`?

Reply:
For me a toggle of playing is clearer when using the old code.
But otherwise I like the new entity layer system.